workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/SetKinematicFromAuthoritySystem.cs

```diff
@@ -27,16 +27,20 @@ private struct NewEntityData
             public readonly int Length;
             public ComponentArray<Rigidbody> Rigidbody;
 
-            // If authority is gained on the first tick there will be an auth changed component
+            // If authority is not gained on the first tick there will be no auth changed component
             public SubtractiveComponent<Authoritative<TransformInternal.Component>> DenotesNotAuthoritative;
+            public ComponentDataArray<KinematicStateWhenAuth> KinematicStateWhenAuth;
             [ReadOnly] public ComponentDataArray<NewlyAddedSpatialOSEntity> DenotesNewEntity;
+            [ReadOnly] public ComponentDataArray<ManageKinematicOnAuthorityChangeTag> DenotesShouldManageRigidbody;
```

Comment:
Given we defer adding the components in the Monobeavhiour by a frame - how does this ever fire?

Reply:
Fixed by removing the deferral.